service/src/main/java/uk/gov/hmcts/reform/fpl/model/Other.java

```diff
@@ -46,4 +53,64 @@ public void addRepresentative(UUID representativeId) {
     public boolean containsConfidentialDetails() {
         return "Yes".equals(detailsHidden);
     }
+
+    @Data
+    @EqualsAndHashCode(callSuper = true)
+    private static class OtherParty extends Party {
+
+        @Builder
+        private OtherParty(String partyId,
+                           PartyType partyType,
+                           String firstName,
+                           String lastName,
+                           String organisationName,
+                           LocalDate dateOfBirth,
+                           Address address,
+                           EmailAddress email,
+                           Telephone telephoneNumber) {
+            super(partyId, partyType, firstName, lastName, organisationName, dateOfBirth, address, email,
+                telephoneNumber);
+        }
+    }
+
+    @JsonIgnore
+    @Override
+    public Party getConfidentialParty() {
+        return OtherParty.builder()
+            .firstName(this.getName())
+            .address(this.getAddress())
+            .telephoneNumber(Telephone.builder().telephoneNumber(this.telephone).build())
+            .build();
+    }
+
+    @JsonIgnore
+    @Override
+    public Other getConfidentialDetails() {
+        return Other.builder()
+            .name(this.name)
+            .address(this.address)
+            .telephone(this.telephone)
+            .build();
+    }
+
+    @JsonIgnore
```

Comment:
Don't think you need this here as I don't think jackson would look for a method beginning with add